docker/docker-compose.yml

```diff
@@ -23,8 +23,12 @@ services:
       # Satellite authentication/authorization support
       - JWT_ISSUER
       - JWT_AUDIENCE
+<<<<<<< HEAD
```

Comment:
This was left from the rebase

Reply:
Surprisingly, I didn't see this one when reviewing it